### PR TITLE
Create 2016-06-02-TGIF-86-TBD.md

### PR DESCRIPTION
**Pull request by skeskali**

### CODE REVIEW

File: _posts/2016-06-02-progressive-pronouns-secret-dance-off tgif-86.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+
+## Open Source
+
+[SQLite: The Art of Keep It Simple](http://www.jarchitect.com/Blog/?p=2392)
+
+## Design
+
+[Karolina Szczur - Platform for Belonging(HybridConf 2015)](https://vimeo.com/166454411)
```
Comment (gr2m) [inline _posts/2016-06-02-progressive-pronouns-secret-dance-off tgif-86.md]: I think I’d move this to `The Tech World and Culture`

**Comment by gr2m** [comment]
can we add a link to Stephanie’s short story? https://twitter.com/radiomorillo/status/738759089317117956

**Comment by gr2m** [comment]
we can add https://ia.net/know-how/on-icons to design

**Comment by jennwrites** [comment]
LGTM

**Pull request merged**